File: Lab3/Lab3/Model/Account.cs
using System.Text.Json.Serialization;

namespace Lab3.Model
{
    class Account
    {
        [JsonPropertyName("id")]
        public int Id { get; set; }

        [JsonPropertyName("money")]
        public long Money { get; set; }

        [JsonPropertyName("deletionTime")]
        public string DeletionTime { get; set; }

        public override string ToString()
        {
            return $"[[id]: {Id}; [money]: {Money}; [deletionTime]: {DeletionTime}]";
        }
    }
}

File: Lab3/Java/Lab3/src/model/Account.java
package model;

import com.google.gson.annotations.SerializedName;

public class Account {
    @SerializedName("id")
    private int id;

    @SerializedName("money")
    private long money;

    @SerializedName("deletionTime")
    private String deletionTime;

    public Account(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public long getMoney() {
        return money;
    }

    public void setMoney(long money) {
        this.money = money;
    }

    public String getDeletionTime() {
        return deletionTime;
    }

    public void setDeletionTime(String deletionTime) {
        this.deletionTime = deletionTime;
    }

    @Override
    public String toString() {
        return String.format("{[id]: %s; [money]: %s; [deletionTime]: %s}", id, money, deletionTime);
    }
}
